SimpleMap/ApiCall.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 
 func fetchData() async throws -> [Location] {
-	let url = URL(string: "http://35.179.12.156/get/list/coords")!
+	let url = URL(string: "http://13.42.62.23/get/list/coords")!
```

Comment:
in swift it's generally not encouraged to use force unwrap pretty much anywhere in production code, instead you want to handle optionals in a fail-safe way. Try rewriting this to use e.g guard statements https://www.hackingwithswift.com/new-syntax-swift-2-guard